source/fundamentals/enterprise-auth.txt

```diff
@@ -478,6 +478,71 @@ callback function that you defined:
    :end-before: end-credential-callback
    :emphasize-lines: 6
 
+Kubernetes
+~~~~~~~~~~
+
+If your application runs on a Kubernetes cluster, you can authenticate to MongoDB 
+by using the {+driver-short+}'s built-in Kubernetes support.
```

Comment:
The "k8s" OIDC integration specifically targets AWS EKS, Azure AKS, GCP GKE, or any Kubernetes clusters with a configured [ServiceAccount](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#bound-service-account-token-volume). Should we add those details to this section? Is there documentation that provides more details on the general drivers Kuberenets OIDC support that we can link to?

Reply:
I added a link to this "Managing Service Accounts" guide you mentioned in the intro of this Kubernetes section. There is no general guide yet for Kubernetes OIDC support. Let me know what you think, thanks!